app/views/tasks/_table.html.slim

```diff
@@ -2,55 +2,51 @@ table
   caption
     = caption
   thead
-    - %w[task title size point_of_contact started_on expected_on expected_duration].each do |header|
+    th &check;
```

Comment:
What is this syntax? I can't figure it out.

Reply:
It's an html entity and some of them have nice shorthands
https://www.compart.com/en/unicode/U+2713

Reply:
ahhhh cool cool. Thanks!